projects/pulsar_lvds_adc/zed/system_project.tcl

```diff
@@ -12,32 +12,31 @@ source $ad_hdl_dir/projects/scripts/adi_board.tcl
 # LEGEND: 18 BITS RESOLUTION AD7960 - 0
 #         16 BITS RESOLUTION AD7626 - 1
 
-set RESOLUTION_16_18N 0
-
-if {[info exists ::env(RESOLUTION_16_18N)]} {
-  set RESOLUTION_16_18N $::env(RESOLUTION_16_18N)
-} else {
-  set env(RESOLUTION_16_18N) $RESOLUTION_16_18N
-}
+set RESOLUTION_16_18N [get_env_param RESOLUTION_16_18N 0]
 
 adi_project pulsar_lvds_adc_zed 0 [list \
-  RESOLUTION_16_18N  $RESOLUTION_16_18N \
+  RESOLUTION_16_18N [get_env_param RESOLUTION_16_18N  1] \
```

Comment:
Is it 1 or 0 the default configuration?
It should be corrected in these 2 lines and it should be written in the comments as well.

Reply:
it s 0 (fixed). do we have an example on how it should appear in the comments?

Reply:
Sadly, no, but the READMEs will have (coming in the following week).